src/course-unit/CourseUnit.test.jsx

```diff
@@ -91,19 +89,18 @@ describe('<CourseUnit />', () => {
     const { open } = window;
     window.open = jest.fn();
     const { getByRole } = render(<RootWrapper />);
+    const { draft_preview_link, published_preview_link } = courseSectionVerticalMock;
```

Comment:
[nit]: it better to use camelCase style here

Reply:
Agree, changed